x-pack/plugins/security_solution/public/common/components/ml/hooks/use_installed_security_jobs.ts

```diff
@@ -72,3 +72,18 @@ export const useInstalledSecurityJobsIds = () => {
 
   return { jobIds, loading };
 };
+
+export const useInstalledSecurityJobsNamesById = () => {
```

Comment:
```suggestion
export const useInstalledSecurityJobNameById = () => {
```
Nit, this returns a single job name given an id

Reply:
or
```suggestion
export const useInstalledSecurityJobNamesById = () => {
```
think i need to put some logs in here to better understand, maybe ignore me lol